### DISCUSSION TITLE
Change the test contracts names

### DISCUSSION THREAD

**Discussion by andreivladbrg** [sablier-labs/evm-monorepo#134] [General]
I suggest the following changes:
- `SablierV2UnitTest` to `SablierV2BaseTest`
- `SablierV2LinearUnitTest` to `SablierV2LinearTest`
-  `SablierV2Linear__Foo` to `Foo_Tests`
### Example
```
SablierV2BaseTest is Test {}

SablierV2LinearBaseTest is SablierV2BaseTest {}

Foo_Tests is SablierV2LinearBaseTest {}
```
 This way will be cleaner and will make our tests more DRY.

**Comment by andreivladbrg** [comment]
Addressed in this commit: 04b776dba26b3448e8ca50d8743cccf05da91290

**Comment by PaulRBerg** [comment]
@andreivladbrg I think that this has been addressed by https://github.com/sablierhq/v2-core/pull/159.

Locking this discussion.
